### ISSUE TITLE
Default template includes table presentation attributes

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar-event-calendar#15]
https://github.com/jadu/pulsar-event-calendar/blob/master/src/EventCalendar.js#L184

```<table id='event-calendar' class='clndr-table' border='0' cellspacing='0' cellpadding='0'>```

These attributes are being flagged with accessibility testing tools and should be removed.
